src/components/tasks/SuggestionBox/Suggestions.tsx

```diff
@@ -0,0 +1,72 @@
+import { forwardRef } from 'react';
+import classNames from '@/components/tasks/card/card.module.scss';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import { userDataType } from '@/interfaces/user.type';
+import { Loader } from '../card/Loader';
+import SuggestionBox from '../SuggestionBox/SuggestionBox';
+import { SuggestionsProps } from '@/interfaces/suggestionBox.type';
+import { useGetAllUsersByUsernameQuery } from '@/app/services/usersApi';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+
+const Suggestions = forwardRef<HTMLInputElement, SuggestionsProps>(
+    (
+        {
+            assigneeName,
+            searchTerm,
+            showSuggestion,
+            handleAssignment,
+            handleChange,
+            handleClick,
+        },
+        ref
```

Comment:
Why is ref being passed as props ?

Reply:
because i have a ref on input element, but i am accessing it in the parent element. @bharati-21